SampleUsages/Program.cs

```diff
@@ -24,37 +24,37 @@ public new static void Main(string[] args)
 
         #region LambdaTests
         [Command]
-        public void S3Test(string functionName, string blobPath, string srcBucket, string targetBucket, string loadProfile, int eps = 0, bool repeat = false, int durationMinutes = 0)
+        public void S3Test(string functionName, string blobPath, string srcBucket, string targetBucket, string loadProfile, int eps = 0, bool repeat = false, int durationMinutes = 0, int warmUpTimeInMinutes = 3)
```

Comment:
I'm not sure if the default value of this new param should be 3 minutes. It should probably be 0 like durationMinutes. Same comment applies to other methods below.